sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/Region.java

```diff
@@ -32,6 +32,7 @@
     public static final Region CANADA_CENTRAL = new Region("canadacentral", "Canada Central");
     public static final Region CANADA_EAST = new Region("canadaeast", "Canada East");
     public static final Region BRAZIL_SOUTH = new Region("brazilsouth", "Brazil South");
+    public static final Region BRAZIL_SOUTHEAST = new Region("brazilsoutheast", "Brazil Southeast");
```

Comment:
Do we need another `Region.java` in azure-resourcemanager-resources? Can this just use the one in `azure-core-management`?

Reply:
If we add it in core-management, we will do the switch after next release.